1.9/usage/pods/index.md

```diff
@@ -5,11 +5,12 @@ menu_order: 85
 ---
 
 # Overview
-Pods enable you to share storage, networking, and other resources among a group of services on a single agent and address them as one group rather than as separate services, similar to a virtual machine. Pods allow quick, convenient coordination between services that need to work together, for instance a database and web server that make up a content management system. Scaling your pod is also straightforward.
+Pods enable you to share storage, networking, and other resources among a group of applications on a single agent, address them as one group rather than as separate applications, and manage health as a unit.
```

Comment:
I'd split this into 2 sentences. E.g. `...on a single agent. You can then address them...`